deps.ts

```diff
@@ -1,4 +1,4 @@
-import Canvas from 'https://deno.land/x/canvas@v1.2.2/mod.ts';
+import Canvas from "@josefabio/deno-canvas";
```

Comment:
Why not just update this and [deps.ts#L4](https://github.com/Ciaxur/DenoChart/pull/3/files#diff-c5027af4b65d385844cd2ddd712950a06ac0d8db6aec9bccb69ed0c4145fd796R4) to `https://deno.land/x/canvas@v1.4.1/mod.ts'`?

That works for me:
```sh
➜ DenoChart git:(main) ✗ gd deps.ts
diff --git a/deps.ts b/deps.ts
index f8e3dcf..c3d47c2 100644
--- a/deps.ts
+++ b/deps.ts
@@ -1,4 +1,4 @@
-import Canvas from 'https://deno.land/x/canvas@v1.2.2/mod.ts';
+import Canvas from 'https://deno.land/x/canvas@v1.4.1/mod.ts';
 export default Canvas;
 
-export * from 'https://deno.land/x/canvas@v1.2.2/mod.ts';
\ No newline at end of file
+export * from 'https://deno.land/x/canvas@v1.4.1/mod.ts';

➜ DenoChart git:(main) ✗ deno --version
deno 1.33.2 (release, x86_64-unknown-linux-gnu)
v8 11.4.183.1
typescript 5.0.3
➜ DenoChart git:(main) ✗ deno run ../main.ts 
Initialized Canvas Instance W[720] H[480]
Create Graph with Options: {
  height: 480,
  width: 720,
  backgroundColor: { r: 0, g: 0, b: 0, a: 0.75 },
  titleText: "Uptime",
  xAxisText: "Hours",
  yAxisText: "Day",
  yMax: 50,
  yPadding: 0,
  xPadding: 0,
  bar_width: 25,
  bar_spacing: 5,
  graphSegments_X: 18,
  graphSegments_Y: 10,
  titleColor: "rgb(255,255,255)",
  xTextColor: "rgba(255,255,255,1)",
  yTextColor: "rgba(255,255,255,1)",
  xSegmentColor: "rgba(255,255,255,0.5)",
  ySegmentColor: "rgba(255,255,255,0.5)",
  graphValuePrecision: 2,
  verbose: true
}
yMax Evaluated to:  50
✅ Granted write access to "image.png".
Graph save to 'image.png'
```